guide/english/certifications/javascript-algorithms-and-data-structures/functional-programming/use-higher-order-functions-map-filter-or-reduce-to-solve-a-complex-problem/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Write Higher Order Arrow Functions
+title: Use Higher-Order Functions map, filter, or reduce to Solve a Complex Problem 
```

Comment:
```suggestion
title: Use Higher-Order Functions to Solve a Complex Problem
```
If we go ahead with these 3, we'll also need to change the file names

Reply:
See my comment above.